modules/core/src/models/core.ts

```diff
@@ -209,22 +209,3 @@ export interface SVGRLoadedInterface {
      */
     default: string;
 }
-
-export interface LegacyModeInterface {
-    apiResourcesV1: boolean;
-    apiResourcesV2: boolean;
-    applicationListSystemApps: boolean;
-    applicationOIDCSubjectIdentifier: boolean;
-    applicationRequestPathAuthentication: boolean;
-    applicationSystemAppsSettings: boolean;
-    approvals: boolean;
-    certificates: boolean;
-    backupCodesForSubOrganizations: boolean;
-    loginAndRegistrationEmailDomainDiscovery: boolean;
-    organizations: boolean;
-    rolesV1: boolean;
-    roleMapping: boolean;
```

Comment:
Better to keep the codes related to this config since we are supporting this feature for migrating applications. Don't need to be under the legacy configs. But we will need to maintain a config to show that UI.

Reply:
Ack